pallets/tendermint-light-client/src/utils.rs

```diff
@@ -90,3 +90,16 @@ where
     let bytes = base64::decode(&string).map_err(serde::de::Error::custom)?;
     Ok(H256::from_slice(&bytes))
 }
+
+/// Maps Tendermint's hash to H256 type.
+/// Since Tendermint hash type has a `None` variant and Substrate uses
+/// a fixed-size array for this, function has to return `Option` to faithfully
+/// represent the original value.
+///
+/// Returns `None` if input value is `None` variant.
+pub fn tm_hash_to_hash(tm_hash: &Hash) -> Option<H256> {
```

Comment:
```suggestion
pub fn tendermint_hash_to_h256tm_hash: &Hash) -> Option<H256> {
```

Reply:
but H256 is not Tendermint's type. I'm using `tm_*` as suffix for types that come from Tendermint's crate.

Reply:
Renamed in [`be802d4` (#361)](https://github.com/Cardinal-Cryptography/aleph-node/pull/361/commits/be802d4037cc173e256b0fe174884799b6fc53c0)